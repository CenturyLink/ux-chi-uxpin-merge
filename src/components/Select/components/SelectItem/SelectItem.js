import PropTypes from 'prop-types';
import React from 'react';

/**
 * @uxpinnamespace Select
 */
export default function SelectItem(props) {
  return (
    <span>
      {props.children}
    </span>
  );
}

SelectItem.propTypes = {
  /** @uxpinpropname Label */
  children: PropTypes.string,
};
