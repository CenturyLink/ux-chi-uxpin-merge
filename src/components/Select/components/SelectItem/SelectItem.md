# SelectItem

```javascript
<SelectItem>Option 1</SelectItem>
```