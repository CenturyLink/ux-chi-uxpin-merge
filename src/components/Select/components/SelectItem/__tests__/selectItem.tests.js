import React from "react";
import * as enzyme from "enzyme";
import toJson from "enzyme-to-json";
import SelectItem from "../SelectItem";

test("renders SelectItem correctly", () => {
  const selectItem = (
      <SelectItem>Option 1</SelectItem>
  );

  expect(toJson(enzyme.shallow(selectItem))).toMatchSnapshot(`enzyme.shallow`);
});