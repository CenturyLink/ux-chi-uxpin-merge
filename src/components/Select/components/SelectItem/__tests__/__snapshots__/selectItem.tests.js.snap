// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SelectItem correctly: enzyme.shallow 1`] = `
<span>
  Option 1
</span>
`;
